Icon/Icon.js

```diff
@@ -50,7 +49,8 @@ const IconBase = kind({
 		 * @type {String|Object}
 		 * @public
 		 */
-		children: PropTypes.oneOfType([PropTypes.string, PropTypes.object])
+		children: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+		css: PropTypes.object
```

Comment:
can you add a short description for this prop?
Also, keep an empty line between props